src/util/string.h

```diff
@@ -387,12 +387,12 @@ inline void str_replace(std::string &str, const std::string &pattern,
 }
 
 /**
- * Remove all chat escape sequences in \p s.
+ * Remove all escape sequences in \p s.
  *
  * @param s The string in which to remove escape sequences.
  * @return \p s, with escape sequences removed.
  */
-std::wstring removeChatEscapes(const std::wstring &s);
+std::wstring removeEscapes(const std::wstring &s);
```

Comment:
This function should be lower_case_underscore style since it's a standalone function.